### PR TITLE
Added plugin octoprint_eta

### PR DESCRIPTION
**Pull request by pablogventura**
A plugin to display finish time for current print.

### CODE REVIEW

File: _plugins/octoprint-eta.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+layout: plugin
+
+id: display-eta
+title: OctoPrint-Display-ETA
+description: Show finish time (ETA time) for current print.
+author: Pablo Ventura
+license: AGPLv3
+
+# TODO
+date: today's date in format YYYY-MM-DD, e.g. 2015-04-21
```
Comment (kantlivelong) [inline _plugins/octoprint-eta.md]: Needs correcting

File: _plugins/octoprint-eta.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+homepage: https://github.com/pablogventura/Octoprint-ETA
+source: https://github.com/pablogventura/Octoprint-ETA
+archive: https://github.com/pablogventura/Octoprint-ETA/archive/master.zip
+
+# TODO
```
Comment (kantlivelong) [inline _plugins/octoprint-eta.md]: Can you clean up all of these TODO/comments?

**Comment by kantlivelong** [comment]
Thanks for the submission.

A few things:

1. Can you review my comments on the changes submitted?
2. In your [`__init__.py`](https://github.com/pablogventura/Octoprint-ETA/blob/master/octoprint_eta/__init__.py) can you add a valid `__plugin_description__`?
3. I would also highly encourage that you add the [softwareupdate check hook](http://docs.octoprint.org/en/master/bundledplugins/softwareupdate.html#octoprint-plugin-softwareupdate-check-config) so that users can receive updates.

**Comment by pablogventura** [comment]
Thanks for your help! Here is the corrected version.

**Comment by kantlivelong** [comment]
Looks good now. Merging.

**Comment by pablogventura** [comment]
Thanks!

**Pull request merged**